example/example.dart

```diff
@@ -2,6 +2,8 @@ import 'dart:developer';
 
 import 'package:flutter/material.dart';
 import 'package:flutter_gyw/flutter_gyw.dart';
+import 'package:flutter_gyw/src/fonts.dart';
+import 'package:flutter_gyw/src/icons.dart';
```

Comment:
You can't do that !
The example is supposed to use only the public API

Reply:
Whoops. I forgot to export the enums so my IDE decided to import from the private API.